xla/service/cpu/runtime_single_threaded_fft.cc

```diff
@@ -18,11 +18,11 @@ limitations under the License.
 #include "absl/base/dynamic_annotations.h"
 #include "xla/service/cpu/runtime_fft_impl.h"
 
-ABSL_ATTRIBUTE_NO_SANITIZE_MEMORY void __xla_cpu_runtime_EigenSingleThreadedFft(
+ABSL_ATTRIBUTE_NO_SANITIZE_MEMORY void __xla_cpu_runtime_DuccSingleThreadedFft(
```

Comment:
You may want to remove the "SingleThreaded" part of this name.  Ideally we wouldn't need to rename the symbol yet again when it becomes multithreaded.  There won't be much difference in the implementation other than setting the active threadpool.

Reply:
I guess it doesn't matter. We should pass something that impl ducc's threadpool class to the FftImpl, so here I will pass a pseudo_threadpool.

Reply:
Hmm, sorry, I understand now.

Where should we `set_active_threadpool` in the runtime? Since if as you suggest, we will not inline the single-threadedness into the IR code, how will we encode the single/multi-threadedness?